executor.sh

```diff
@@ -1,12 +1,14 @@
 declare -a os=('u12' 'u14')
-declare -a languages=('' 'nod' 'pyt' 'php' 'rub' 'gol' 'clo' 'jav' 'sca')
+declare -a languages=('' 'nod')
 declare -a languageVersions=('' 'pls' 'all')
 
+imageTag="prod"
```

Comment:
Now using the `imageTag` so that it is easy to determine which tag we want to build and push.